app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/MainFragment.kt

```diff
@@ -16,16 +17,16 @@ class MainFragment : Fragment() {
     private lateinit var viewModel: MainViewModel
 
     override fun onCreateView(
-        inflater: LayoutInflater, container: ViewGroup?,
+        inflater: LayoutInflater,
+        container: ViewGroup?,
```

Comment:
Suggestion 3: Slightly altered the layout of the onCreateView function to be more consistent and readable.
Source: https://kotlinlang.org/docs/coding-conventions.html#class-headers